samples/matter/common/src/diagnostic/diagnostic_logs_crash.h

```diff
@@ -81,6 +86,11 @@ class CrashData {
 	 */
 	size_t CalculateSize();
 
+	CHIP_ERROR GetCrashLogs(chip::MutableByteSpan &outBuffer, bool &outIsEndOfLog);
+	CHIP_ERROR FinishCrashLogs();
+	size_t GetCrashLogsSize();
+	CHIP_ERROR MoveCrashLogsToNVS();
+
```

Comment:
They are public methods so maybe it is worth adding descriptions to them.

Reply:
Done